server/README.md

```diff
@@ -381,8 +381,8 @@ Keycloak image allows you to specify both a private key and a certificate for se
 
 Those files need to be mounted in `/etc/x509/https` directory. The image will automatically convert them into a Java keystore and reconfigure Wildfly to use it.
 
-It is also possible to provide an additional CA bundle and setup Mutual TLS this way. In that case, you need to mount an additional volume to the image
-containing a `crt` file and point `X509_CA_BUNDLE` environmental variable to that file.
+It is also possible to provide an additional CA bundle and setup Mutual TLS this way. In that case, you need to mount an additional volume (or multiple volumes) to the image the contains all necessary `crt` files. The final step is to point `X509_CA_BUNDLE`
+ environmental variable to them using space (` `) a separator. In case of an OpenShift environment, that could be `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt /var/run/secrets/kubernetes.io/serviceaccount/ca.crt`.
```

Comment:
```s/The final step is to point `X509_CA_BUNDLE` environmental variable to them using space (` `) a separator./The final step is to configure the `X509_CA_BUNDLE` environment variable to contain a list of the locations of the various CA certificate bundle files specified before, separated by space (` `)./g```

```s/In case of an OpenShift environment, that could be ..../For example, in the case of an OpenShift environment, the `X509_CA_BUNDLE` value would be .../g```

Reply:
Fixed thanks!